src/orga/governance.rst

```diff
@@ -85,13 +85,19 @@ Standard items are ones where public record and discourse is
 preferable. Sensitive voting items are ones where the results of the
 vote should remain private to the voters after the vote has occurred.
 Sensitive votes should take place on a secure anonymous voting platform
-(such as `Polys <polys.me>`_ or
-`the Helios voting system <https://vote.heliosvoting.org/>`_) in order to
-retain anonymity.
+in order to retain election integrity and anonymity. (We have used `Polys <polys.me>`_ 
+and `the Helios voting system <https://vote.heliosvoting.org/>`_, but are 
+open to any secure, anonymous system.) The email capability of your chosen 
+voting platform should be used for sending voting invitations and reminders,
+and you should use the email list from
+https://github.com/conda-forge/conda-forge.github.io/blob/master/src/core.csv as
+the authoritative list of emails to use.
 
 The default voting period is 1 week (7 days). This may be modified at
 the time when a vote is called, but may never be less than 24 hrs.
 
+Additional requirements may apply in case low turnouts have to be handled, see "Quorum" below.
```

Comment:
```suggestion
Additional requirements may apply in case low turnouts have to be handled, see "Quorum" below.
For sensitive votes, this must be planned ahead when deciding the duration of the vote.
```

Reply:
Why the restriction on "sensitive" votes?

Reply:
Other votes can be extended. Sensitive votes have to be redone.

Reply:
Hm.. Not really, actually. The document only talks about vote end dates but does not offer the possibility the change/extend those.
De facto we just have *blatantly ignored the end dates* on many occasions -- something I don't like and would like us to change in the future. It's just not good practice and does not seem professional.
If we want to make use of an extended voting period, we should just open the vote for a month instead of two weeks then, IMHO. (Otherwise we should amend the governance document to clarify what "end date" is supposed to mean for standard votes then.)